.circleci/smoke_test/docker/Dockerfile

```diff
@@ -25,12 +26,16 @@ RUN apt-get -qq update && apt-get -qq -y install curl bzip2 sox libsox-dev libso
 
 ENV PATH /opt/conda/bin:$PATH
 
-RUN conda create -y --name python3.6 python=3.6
+
 RUN conda create -y --name python3.7 python=3.7
 RUN conda create -y --name python3.8 python=3.8
+RUN conda create -y --name python3.9 python=3.9
+RUN conda create -y --name python3.10 python=3.10
+
 SHELL [ "/bin/bash", "-c" ]
 RUN echo "source /usr/local/etc/profile.d/conda.sh" >> ~/.bashrc
-RUN source /usr/local/etc/profile.d/conda.sh && conda activate python3.6 && conda install -y -c conda-forge sox && conda install -y numpy
 RUN source /usr/local/etc/profile.d/conda.sh && conda activate python3.7 && conda install -y -c conda-forge sox && conda install -y numpy
 RUN source /usr/local/etc/profile.d/conda.sh && conda activate python3.8 && conda install -y -c conda-forge sox && conda install -y numpy
+RUN source /usr/local/etc/profile.d/conda.sh && conda activate python3.9 && conda install -y -c conda-forge sox && conda install -y numpy
+RUN source /usr/local/etc/profile.d/conda.sh && conda activate python3.10 && conda install -y -c conda-forge sox && conda install -y numpy
```

Comment:
I think this is based off of torchaudio's Dockerfile and `sox` is not required in text. (technically it's no longer required in audio as well).